src/sentry/workflow_engine/migrations/0014_aci_model_updates_from_milestones.py

```diff
@@ -0,0 +1,175 @@
+    # of a table.
+    # Here are some things that make sense to mark as post deployment:
+    # - Large data migrations. Typically we want these to be run manually so that they can be
+    #   monitored and not block the deploy for a long period of time while they run.
+    # - Adding indexes to large tables. Since this can take a long time, we'd generally prefer to
+    #   run this outside deployments so that we don't block them. Note that while adding an index
+    #   is a schema change, it's completely safe to run the operation after the code has deployed.
+    # Once deployed, run these manually via: https://develop.sentry.dev/database-migrations/#migration-deployment
+
+    is_post_deployment = False
+
+    dependencies = [
+        ("sentry", "0792_add_unique_index_apiauthorization"),
+        ("workflow_engine", "0013_related_name_conditions_on_dcg"),
+    ]
+
+    operations = [
+        migrations.SeparateDatabaseAndState(
+            database_operations=[
+                migrations.RunSQL(
+                    """
+                    ALTER TABLE "workflow_engine_detector" ADD COLUMN "config" boolean NOT NULL DEFAULT true;
+                    """,
+                    reverse_sql="""
+                    ALTER TABLE "workflow_engine_detector" DROP COLUMN "config";
+                    """,
+                    hints={"tables": ["workflow_engine_detector"]},
+                ),
+            ],
+            state_operations=[
+                migrations.AddField(
+                    model_name="detector",
+                    name="config",
+                    field=models.JSONField(blank=True, default=dict, null=True),
+                ),
+            ],
+        ),
+        migrations.AddField(
+            model_name="detector",
+            name="created_by",
+            field=sentry.db.models.fields.hybrid_cloud_foreign_key.HybridCloudForeignKey(
+                "sentry.User", db_index=True, null=True, on_delete="SET_NULL"
+            ),
+        ),
+        migrations.AddField(
+            model_name="detector",
+            name="description",
+            field=models.TextField(blank=True, null=True),
+        ),
+        migrations.SeparateDatabaseAndState(
+            database_operations=[
+                migrations.RunSQL(
+                    """
+                    ALTER TABLE "workflow_engine_detector" ADD COLUMN "enabled" boolean NOT NULL DEFAULT true;
+                    """,
+                    reverse_sql="""
+                    ALTER TABLE "workflow_engine_detector" DROP COLUMN "enabled";
+                    """,
+                    hints={"tables": ["workflow_engine_detector"]},
+                ),
+            ],
+            state_operations=[
+                migrations.AddField(
+                    model_name="detector",
+                    name="enabled",
+                    field=models.BooleanField(default=True),
+                ),
+            ],
```

Comment:
Same comment from before - you can define this with `db_default=True` and not have the sql